client/src/components/SideInfoPanel/SideInfoPanel.test.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import { ThemeProvider } from 'styled-components';
+import GamesContext from '../../context';
+import Theme from '../styles/theme';
+import SideInfoPanel from './SideInfoPanel';
+
+const renderSideInfoPanel = (sidePanelImg, sidePanelInfo, setModal, recentReviews, allReviews) => {
```

Comment:
You might be able to write this component more simply if I'm not mistaken since its stateless?

So instead of:

const ComponentName = (props) => {
  return render(
    //jsx code
  )
}

You could just write:

const ComponentName = (props) => (
  //jsx code
)

Overall though, solid component - easy to read